relay-base-schema/src/data_category.rs

```diff
@@ -7,6 +7,11 @@ use serde::{Deserialize, Serialize};
 
 use crate::events::EventType;
 
+/// An error that occurs if a number cannot be converted into a [`DataCategory`].
+#[derive(Debug, PartialEq, thiserror::Error)]
+#[error("number could not be converted into a [`DataCategory`].")]
```

Comment:
```suggestion
#[error("Unknown numeric data category {0} can not be converted into a DataCategory.")]
```

Doc links like this don't really work in error messages, also always nice if the error message contains the cause, not sure if my suggestion is optimal, but something along these lines.

Reply:
You're right, that was an oversight of just moving documentation around.